index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+
+<title>Emelio's Site</title>
+<link rel="stylesheet" href="style.css"/>
+
+
+<header>
+  <h1 id="header"> Emilio Comiling</h1>
+  <img class= "image" src="emelio.jpg">
+  <h2>About Emelio </h2>
+  
+  <p class="text">
+    Originally from the Philippines, Emelio moved over to the England 12 years ago and has settled in nicely, devoting his hours to working in healthcare, looking after patients with spinal injuries. 
+    Although he is quite new to the tech world, Emelio considers himself to be quite tech savvy and enjoys the challenges that come along with coding. 
+  </p>
+  </header>
+<br>
+<br>
+
+
+<body> 
+ <h3>What Does Emelio Do In His Spare Time?</h3>
+<li>Spend Time With His Kids</li>
+<li>Mountain Biking</li>
+<li>Practising Coding</li>
+</body>
+
+<br>
+
+<aside>
+  <h3>Coding 101</h3>
+  <p>Emelio has been coding for over ten months, all self-taught.
+    Do you want to know how? Here are his go-to sites for learning to code:
+  </p>
+  <ol>
+    <li><a href="https://marksheet.io">Marksheet</a></li>
+    <li><a href="https://www.w3schools.com">W3Schools</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Learn">MDN Web Docs</a></li>
+  </ol>
+
+  <br>
+  <br>
+  <p>
+    <h3>Emelio's Words To Live By:</h3>
+    <blockquote>
+      "Theres always a rainbow after the rain"
+    </blockquote>
+
+
+<footer>
+
+
+<p>
+  Want to connect with Emelio? Just click <a href="https://www.linkedin.com/in/emilio-comiling-393717b4/?originalSubdomain=uk">here</a>.
+</p>
+
+<p> 
+ <input type="text" placeholder="Send Emelio a message">
```

Comment:
Look into how you can make this input a working text field. You could have a form that sends the inputted message as an email.